lib/web_ui/lib/src/engine/semantics/tabs.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'label_and_value.dart';
+import 'semantics.dart';
+
+/// Represents semantic objects that deliver information in a visual manner.
+///
+/// Uses aria tab role to convey this semantic information to the element.
+///
+/// Screen-readers takes advantage of "aria-label" to describe the visual.
```

Comment:
Is it a guarantee that "aria-label" will be used? `label_and_value.dart` may use one of three representations for a label. All representations work with screen readers but they have their own nuances.

Maybe here we should talk about `SemanticsObject.label` instead of aria-label?

Reply:
I copied this from other SemanticsRole. I thought this is describing the use of LabelRepresentation.ariaLabel ?